intel_extension_for_transformers/llm/runtime/graph/__init__.py

```diff
@@ -48,14 +48,14 @@ def __import_package(self, model_name):
             import intel_extension_for_transformers.llm.runtime.graph.opt_cpp as cpp_model
         elif model_name == "bloom":
             import intel_extension_for_transformers.llm.runtime.graph.bloom_cpp as cpp_model
-        elif model_name == "chatglm2":
+        elif model_name == "chatglm":
```

Comment:
please sync with zhenzhong, I believe we support chat glm v1 and v2

Reply:
model_name == "chatglm" or model_name == "chatglm2"

Reply:
fixed